### ISSUE TITLE
Keyboard navigation and focus ring does not work in videos from Learn and Coach

### ISSUE THREAD

**Issue by sairina** [learningequality/kolibri#9174]
## Observed behavior
If a Learner is using the keyboard to navigate to a video, they are unable to see a focus ring appear when they enter the `LearnImmersiveLayout` to view the video - as well as when a Coach is previewing a video in `PlanLessonSelectionContentPreview`:
|Issue in Learn|Issue in Coach|
|--|--|
|<img width="860" alt="Screen Shot 2022-03-11 at 4 13 34 PM" src="https://user-images.githubusercontent.com/13563002/157994534-c3e0e99b-477a-48f6-ae07-23ca3ce87422.png">|<img width="860" alt="Screen Shot 2022-03-11 at 4 35 37 PM" src="https://user-images.githubusercontent.com/13563002/157995712-33ef621f-6cb0-4e70-8d32-068d2e78db34.png">|

## Errors and logs
None

## Expected behavior
A Learner should be able to use the keyboard to access and watch a video assigned to them; a coach should be able to use a keyboard to preview the video.

## User-facing consequences
This is an accessibility issue. When a learner/coach tries to navigate to a video, there are no visual indicators (focus rings) for navigation, nor is the video easily accessible via keyboard nav.

## Steps to reproduce (fully using keyboard navigation in Learn)
1. Learner navigates to their class and then to a lesson
2. Navigate to a video card and presses `Enter`
3. Use the keyboard to try to navigate the video activity
4. Notice that no focus rings appear (though, after enough presses on `Tab`, you will find that the focus ring will appear in the activity bar.

In coach (using keyboard navigation):
1. Navigate to the "Plan" a lesson tab
2. Under "Lessons", open a lesson with a video, or create a lesson with a video
3. Navigate to a video and press `Return`
4. When the video preview has opened, notice that the video will play automatically, and the user can navigate through the media player - but the focus ring is not trapped within the window, which makes it very difficult (many key presses) to get to the "go back" icon (which is a larger issue for the whole application and will be documented in another issue).

## Context
Kolibri 0.15
MacOS
Chrome

**Comment by rtibbles** [comment]
Does the same thing happen if you are previewing a video in the Coach plugin, during the lesson creation flow? Would help to show if this is a universal issue with the media player plugin, or a specific interaction in the Learn plugin.

**Comment by sairina** [comment]
Ah, yes, it is also in Coach. I'll update that.

**Comment by sairina** [comment]
(But note that the problem in coach seems to be slightly different)

**Comment by marcellamaki** [comment]
I think that the learn issue might be settings related, or could be related to https://github.com/learningequality/kolibri/issues/6035. After doing a little bit of testing, it would seem as though the focus defaults to the video player controls, but since the focus within the player doesn't use our focus outline, it becomes hard to track the focus. But, I am able to navigate through all of the options (side panel issues aside) with tabbing.

As for the question about the _number_ of tab presses... it seems like using shift + Tab ("backwards tab order") will allow a quick navigation back out of the video player into the top bar.

@radinamatic -- are you aware of any recommendations or best practices regarding these immersive style pages and whether they should behave like a modal (trapping the focus entirely) or more like a regular page, where the user can tab into other tabs open in the browser, the URL, etc.? I'm not sure what the better option is here.

**Comment by marcellamaki** [comment]
Closed with https://github.com/learningequality/kolibri-design-system/pull/185
